CONTRIBUTING.md

```diff
@@ -24,6 +24,13 @@
 
 * Ask any question about how to use GCCRS in [Zulip](https://gcc-rust.zulipchat.com/).
 
+### ** PR Policy **
+
+* The PR policy: Everything has to go through a PR
+  - An exception to this rule will be the merge commits of updating the repo against upstream GCC
+* Reviewers/Maintainers of the project (aka people who have bors rights) should be pinged for reviews/questions. 
```

Comment:
Catch-22: The only people who can see this list are people who have bors rights.

Reply:
I would add :
```
* Changes must be rebased (no merge commit) over latest `master` branch and be reasonably split
* During the review process, new changes should not be forced-pushed except for the last grooming step
```

Reply:
Force pushing isn't really the problem. If you rebase a PR on top of master you'll have to force push, but the commits still exist in the previous order. The problem is, if you only have one commit in a PR and every change is amended to that commit and then force pushed.

I would write the second point as 

```
* During the development/review process, changes to the PR should be pushed as new commits and only squashed together once the PR is ready to get merged.
```